localstack/services/apigateway/apigateway_listener.py

```diff
@@ -14,6 +14,11 @@
 from localstack.services.kinesis import kinesis_listener
 from localstack.services.generic_proxy import ProxyListener
 
+try:
```

Comment:
I don't think this is a good way, but it's the only one worked for me while handling backward compatibility for `urllib` (didn't use python for long time though). We can also not rely on libraries at all if this not the perfect way.

Reply:
I tried this locally but didn't work (maybe my setup was corrupted) anyways I changed  it and lets see if the build passes! Thank you 👍